ext/web/06_streams.js

```diff
@@ -4585,6 +4604,7 @@
   }
 
   const _resourceBacking = Symbol("[[resourceBacking]]");
+  const _resourceBackingUnrefable = Symbol("[[resourceBackingUnrefable]]");
```

Comment:
This distinction exists to prevent unrefable streams being used in regular fast streams that are unaware of refability? Please add a comment

Reply:
Correct, added the comment.